qiskit/tools/apps/optimization.py

```diff
@@ -298,7 +298,7 @@ def eval_hamiltonian(Q_program, hamiltonian, input_circuit, shots, device):
             quantum_state = result.get_data(circuit[0]).get('quantum_state')
             if quantum_state is None:
                 quantum_state = result.get_data(
-                    circuit[0]).get('quantum_states')
+                    circuit[0]).get('quantum_state')
```

Comment:
should we change output of statevector_simulators to be `statevector` instead of `quantum_state`

Reply:
let's do this in the qobj PR. we have to make sure some other stuff are in line with qobj as well, and i think we can do that after the schemas are decided.

Reply:
i just went ahead and did this.